### DISCUSSION TITLE
The final balance thread before 1.0

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#11148] [Unstable feedback]
In no particular order, a summary of all things that could use some balancing before 1.0.
I will try to keep it as short as possible, so no long detailled explanations. Suggestions in italic
I'll keep this updated, so discuss

----------

# Resources

Stealing is the source of most of resource problems. #10621
Here are some resource outliers:

- Aluminium
Exceptionally common and easy to get due to stealing, no proper "sink" for alu (ammo boxes can be recycled).

- Plastic
Exceptionally common and easy to get due to stealing and basic gear deconstruction (found on every dead human)
_Reduce headset yield from 2 plastic to 1 silicon_

- Lead
Has an extremely high use rate, and is somewhat scarce in nature, so much that piercing/shredder/explosive ammo is easier to make. Deconstructing coilgun boxes the primary source of Lead.
_Remove Lead from small ammunition recipes (see Weapons), reduce coilgun ammo boxes sold in military outposts from 15 to 3

- Chlorine
Used in all poisons, only exists in 1 mineral, but dozens can be bought for 8mk each at mining outpost.

- Calcium
One of the least used resources but abundant due to Mudraptor loot
_Change Tonic Liquid recipe to use more Calcium but less Zinc/Potassium_

- Dementonite
Sees very little use with tools gone. 
_Merge into sulphurite? Add recipes?_

# Submarine Weapons

All submarine weapons except the Railgun use way too little power. 
A 70% supercap charge rate will often out-charge a gun firing, supercaps can be upgraded beyond that.
See #9620

- Coilgun/Double
Explosive ammo is exceptionally cheap and excessively powerful against monsters, humans and subs alike #10063. Also easier to make than regular ammo (due to lead)
_Increase the cost significantly, maybe 2x C-4_

- Chaingun
Balanced, Shredder ammo easier to make than regular ammo (due to Lead)

- Railgun
A resource sink, regular shells/canisters could use a damage buff considering the resource cost

- Flak Canon
The cheapest ammunition in the game. 25 shots per box is a LOT
See #10865

# Handheld weapons

Most weapons pre-talent rework got power-creeped, and it is causing balance issues
Stunlock can be a big problem, as damage modifiers increase so does stun
Ammunition recipes use a LOT of Lead

- Revolver, SMG, Shotgun, Machine Pistol:
Early game weapons that still use almost as much Lead in their recipe as the stronger, new weapons.
_Decrease Lead/Magnesium cost or replace Lead with Copper in ammunition recipes_

- Grenade launcher
The rifle grenade has made the grenade launcher obsolete. GL isn't craftable and you can't get Masterwork bonuses unlike rifle
_Increase GL damage or add a cooldown to shooting grenades (change reload speed on containing)_

- Handcanon:
Stunlocks completely after the first shot

- Ceremonial sword:
The outlier among melee weapons, the higher DPS bar none

- Riot shield:
Fun, but not useful as it is very hard to shoot over it.
_1-handed riot shield_

# Talents

Discussed at length here #10165. You can find a "tier list" of sorts here https://imgur.com/a/CtOriqL
The outliers are:

- Captain's skipper tree giving massive, passive buffs for the submarine (30k+ worth of upgrades) and crew (more repair speed than hardened tools). _Old Inspire was better in that regard_. 

- Security's rifle tree bonuses are a bit too strong damage wise (the weapons are already powerful)

- Medic has a lot of weak/useless passives, especially in T1. Some fun but weak items (Acid, Pressure stabilizer)

- Mechanic's Exosuit singlehandedly undoes every health balance changes, extreme damage reduction values and massive inventory space. _Nerf both, make exosuit use Uranium rods just like FB3000_
 
- Assistant talents have the biggest power disparity, either giving massive passive buffs (Layabout/Inspiring Tunes), or really weak (Grayshirt tree)
_Check out https://steamcommunity.com/sharedfiles/filedetails/?id=2926972662_

The stacking of passive talents break the game. Here is pictured an Orca2 with:
+20% engine speed from mechanic, +20% engine speed from captain, +24% engine speed from upgrades
This lets you outrun every creature in the game:
![image](https://user-images.githubusercontent.com/73229309/221968683-af925971-0295-4706-938f-6f86c84808fc.png)

Passive talents let you get more repair speed than items and quality itself (hardened tools vs high morale)

# Other

- Submarine things
Submarines that go over 25km/h tend to outrun most creatures in the game. Permanent or passive sub speed bonuses need to be avoided to prevent that.
Mineral scanner on navigation terminal so early and for so cheap skews the relative scarcity of resources by making it extremely easy to find and harvest minerals early.

- Pirates subs
Enemy submarines tend to be loot pinatas. This is due to each enemy crew having basic gear:
Diving mask, welding tool, headset, one suit per crew. The pirate submarines are also stocked with a lot of supplies
Sub vs Sub engagements are way too short mostly due to the power of explosives, coilgun ammo in particular which is too strong just tears through walls both ways. 
Fights end very quickly, the pirates dont have time to use all their supplies, so you can steal 90% of theirs. Suggest reducing the amount of spare ammunition, removing diving masks from pirate default kit and replacing some diving suits with disposable ones (which will last long enough)

- Abandoned outposts
Currently very easy to "cheese" since they are destructible by default but regular outposts arent. It's very easy to use submarine guns to do most of the work. Outposts are NOT built for being flooded, there is no way to drain water out. This also means bandits have diving masks and welding tool equipped, killing them grants a lot of resources.
Bandits themselves are weak, taking high damage and having low health, and are mostly un-armed
_Give bandits more health/protection, give them machine pistols, make outpost walls invulnerable_

- Ruins
Just like abandoned outposts, ruins can be "cheesed" by cutting holes into the walls using sub guns. Some rooms/walls are more resilient than others, but as a whole you can cut through ruins with explosive ammo. Consider making the walls invulnerable and adding natural entrances to compensate
Ruins also tend to spawn a little too often (always 1 per cave starting 3rd biome), making alien materials not too rare.

- Abyssal monsters
Charybdis and Endworm are currently unable to kill you if you have a suit and stay inside the submarine. The Latcher can at least shoot spikes inside the submarine to hit the crew. Both do not deal enough damage to the crew, so often the submarine just sits on the sea floor until the abyssal is killed. Flooding the submarine is NOT a threat (see Oxygen)
_Increase crew damage, make the abyss floor more hazardous (spawns boarders?)_

- Oxygen
As seen in #9669, the lack of oxygen isn't a threat, outside the sub you can carry 45 minutes of oxygen in one slot, inside the sub you can grab suits on the floor to easily avoid death 

- Plants
Plants have always been able to print resources for free. Their growth was slowed down but there's nothing that prevents crafting a planting a dozen in the same room, which will eventually produce unlimited amounts of Carbon/Sodium.
_Consider making planters not placeable/detachable/craftable, and leave it as a sub building option only_

**Comment by LePiromano** [comment]
so true

**Comment by BiffW** [comment]
The exosuit looks very heavy and character acts very slowly with it. But characters wearing it can still be pushed over by water flow or be thrown into air by a mudraptor, which is hilarious. Consider add some flow resistance(like safety harness) and stun resistance to it.

**Reply by BiffW** [reply]
Plus, I suggest that exosuit with a fulgurium battery equipped should allow the character to move a little faster. Make it apply 0.7 speed multiplier(same as abyss suit). That can make fulgurium battery more attractive.

**Comment by BiffW** [comment]
> * Abandoned outposts
>   Currently very easy to "cheese" since they are destructible by default but regular outposts arent. It's very easy to use submarine guns to do most of the work. Outposts are NOT built for being flooded, there is no way to drain water out. This also means bandits have diving masks and welding tool equipped, killing them grants a lot of resources.
>   Bandits themselves are weak, taking high damage and having low health, and are mostly un-armed
>   _Give bandits more health/protection, give them machine pistols, make outpost walls invulnerable_

machine pistol is too weak. And AI won't hold 2 pistols at same time.
I'd suggest these:
On higher difficulty, give bandits full armor and advanced weapons like auto shotgun and assault rifle or even hmg.
Give them gears as powerful as exosuits and make them invade player's submarine if possible.

**Reply by BiffW** [reply]
Plus, give them depleted fuel ammo and explosive slugs in case player have too much regular resistance with exosuit.

**Reply by BiffW** [reply]
If any one ask why bandits have those powerful gears. You can explain in this way.
They are actually special forces from a hostile faction setting an ambush, disguising the outpost as abandoned to wait for player's arrival.

**Reply by StruggleDuck** [reply]
Only problem is that when you you kill them, you get access to those items.

**Reply by BiffW** [reply]
Limiting these powerful enemies to only appear in high difficulty levels could balance that. So that when players gets there, they should already be able to craft the top gears themselves.
And it can also make losing a high-level bot with specific recipe talent less painful. Because player can still get some talent items from the high-tier enemy mentioned here.

**Comment by duarm** [comment]
> Plants
Plants have always been able to print resources for free. Their growth was slowed down but there's nothing that prevents crafting a planting a dozen in the same room, which will eventually produce unlimited amounts of Carbon/Sodium.
Consider making planters not placeable/detachable/craftable, and leave it as a sub building option only

Carbon and Sodium are not worth selling, so you would only use them for crafting which is a very focused problem, you get so much carbon through other means  t's not really worth wasting pots for that. The only worthwhile use for Pots are for producing Tobbaco in order to sell them at Colonies, which is not that OP  as far as I tried farming tobbacos, you get 5000 for a mission, you need lots of tobbaco to match that. They should make them more interesting, not nerf them even more

**Comment by delizin** [comment]
> Chlorine
Used in all poisons, only exists in 1 mineral, but dozens can be bought for 8mk each at mining outpost.

As the ship doctor, I've been struggling with the lack of chlorine. It is used in so many recipes and in such volume, that it is exceedingly scarce on our sub. I'd really like to use poison as my main weapon to go along with my talents, but it is just unsustainable with the number of enemies we encounter. 

I've been doing a long campaign run with a group of friends in v1.0.8.0 and it seems like a lot of these balance issues are still present and apparent. I think we would be having more fun if all of the issues you identified were fixed.

**Comment by CurePindal** [comment]
sad to see that barely anything of the spot-on analysed items for balancing had been taken into account by the devs.

Do they even review this github section or are we among ourselves here?

**Comment by Hazmat616** [comment]
Copper instead of lead in small arms makes a lot of sense.

**Comment by A2ur3** [comment]
Handheld weapons cause extremely high stun, essentially allowing most of them to stunlock humans

The table below lists the rate of fire per second and stun value for some weapons
and higher quality weapons can even cause longer stun

Weapon         Attack Speed Stun
SMG                 |      5        | 0.195 (Depleted Fuel Magazine) |
Riot Shotgun    |      1        | 0.8  |
Scrap Cannon   |     0.33     | 4.8  |
HMG                 |      10      | 0.1  |
Handcannon    |      1        | 1.5  |
Assault Rifle    |     4.167    | 0.25 |


exosuit's fabrication recipe needs physicorium
